android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt

```diff
@@ -59,6 +59,7 @@ class MultiSelectBottomSheetFragment() : BottomSheetDialogFragment() {
       multiSelectViewModel.saveSelectedLocations(requireContext())
       appMainViewModel.run {
         if (requireContext().isDeviceOnline()) {
+          triggerOneTimeSync()
```

Comment:
```suggestion
         viewModelScope.launch { syncBroadcaster.runOneTimeSync() }
```

Reply:
@ellykits I created a function that addresses the suggested change [here](https://github.com/opensrp/fhircore/blob/a9cd5b56a6464045b1b56fa645c37bfb8f037ad4/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt#L319)